addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -822,6 +844,18 @@ export class Rtc extends Record {
         if (camera) {
             await this.toggleVideo("camera");
         }
+        this.cleanups.push(
+            // only register the beforeunload event if there is a call as FireFox will not place
+            // the pages with beforeunload listeners in the bfcache.
+            subscribe(browser, "beforeunload", (event) => {
+                if (this.store.env.services["multi_tab"]?.isOnLastTab()) {
```

Comment:
No service smuggling :rofl:

Reply:
@tsm-odoo 😆 well it is in the dependencies